functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -32,3 +33,33 @@ def scylla_services_names(db_cluster):
     services = db_cluster.k8s_cluster.kubectl(f"get svc -n {SCYLLA_NAMESPACE} -l scylla/cluster=sct-cluster "
                                               f"-o=custom-columns='NAME:.metadata.name'")
     return [name for name in services.stdout.split() if name not in ('NAME', 'sct-cluster-client')]
+
+
+def rollout_restart(db_cluster: ScyllaPodCluster, namespace: str, deployment: str = None):
+    db_cluster.k8s_cluster.kubectl(f"rollout restart deployment -n {namespace} {deployment if deployment else ''}")
+
+
+def scylla_operator_rollout_restart(db_cluster: ScyllaPodCluster):
+    rollout_restart(db_cluster, namespace=SCYLLA_OPERATOR_NAMESPACE)
+
+
+def get_deployment_rollout_status(db_cluster: ScyllaPodCluster, namespace: str, deployment: str, timeout: str):
+    return db_cluster.k8s_cluster.kubectl(f"rollout status deployment -n {namespace} {deployment} --timeout={timeout}")
+
+
+def wait_for_scylla_operator_rollout_complete(db_cluster: ScyllaPodCluster, timeout: str = '60s'):
+    status = []
+    for deployment in ['scylla-operator', 'webhook-server']:
+        result = get_deployment_rollout_status(db_cluster, namespace=SCYLLA_OPERATOR_NAMESPACE,
+                                               deployment=deployment, timeout=timeout)
+        if f"deployment \"{deployment}\" successfully rolled out" not in result.stdout:
```

Comment:
This approach doesn't look good. This variant is suitable when there is no other ones.
Better to use "-o yaml" or "-o json" option for the CLI command, get yaml or json data, parse it into the python dictionary and read proper status fields.

Reply:
Nothing to parse there:
![Screenshot from 2021-08-03 13-52-48](https://user-images.githubusercontent.com/34435448/128003889-73476757-e636-4567-83e2-55f9de84011b.png)

Also `kubectl rollout status` can't return the answer in the yaml format
![Screenshot from 2021-08-03 13-54-36](https://user-images.githubusercontent.com/34435448/128004100-41248f07-7d21-4c9a-aeea-974d62bb9ba4.png)